docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,392 @@
+
+## Create a package
+Let us build a simple Python HTTP server, which, when it receives an HTTP GET request, responds with the current time at the server. We will start with this and build a package that provides this Python server as a DC/OS service.
+
+
+### Step 1 : Create a simple Python HTTP Server
+For the purposes of this guide, we will be using Python 3 and the HTTPServer module provided by its standard library. Let's create a file called `helloworld.py` in an empty directory called `time-server-service`
+
+```python
+import time
+import http.server
+import os
+
+
+HOST_NAME = '0.0.0.0' # Host name of the HTTP server
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = 8000
+
+class MyHandler(http.server.BaseHTTPRequestHandler):
+    def do_GET(s):
+        """Respond to a GET request."""
+        s.send_response(200)
+        s.send_header("Content-type", "text/html")
+        s.end_headers()
+        s.wfile.write("<html><head><title>Time Server</title></head>")
+        s.wfile.write("<body><p>The current time is {}</p>".format(time.asctime()))
+        s.wfile.write("</body></html>")
+
+if __name__ == '__main__':
+    server_class = http.server.HTTPServer
+    httpd = server_class((HOST_NAME, PORT_NUMBER), MyHandler)
+    print(time.asctime(), "Server Starts - {}:{}".format(HOST_NAME, PORT_NUMBER))
+    try:
+        httpd.serve_forever()
+    except KeyboardInterrupt:
+        pass
+    httpd.server_close()
+    print(time.asctime(), "Server Stops - {}:{}".format(HOST_NAME, PORT_NUMBER))
+```
+
+The code snippet simply starts a Python server and serves the GET requests with HTML that says the current time. You should be able to run this code snippet with `python3 helloworld.py` and browse [localhost:8000](http://localhost:8000).
+
+#### Change port mapping to be dynamic
+
+If we want to get a port number dynamically from available ports, Marathon provides a way to achieve this. We access the available ports using the environment variable `$PORT0`, `$PORT1`, `$PORT2` and so on. This will be explained clearly in a later section. But for now, just change your Python snippet to read the port from an environment variable as below :
+
+```python
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = int(os.environ['PORT0'])
+```
+
+Once you do this, you can browse your server after executing `PORT0=8000 python3 helloworld.py`
+
+### Step 2 : Creating a Docker container
+Creating a Docker container is essential to distribute _this_ service. It runs completely isolated from the host environment by default, only accessing host files and ports if configured to do so. To continue reading, you need to be familiar with Docker; we recommend this [get-started](https://docs.docker.com/get-started/) guide. You should have logged in to your Docker account in your terminal using `docker login`.
```

Comment:
The docker link is also a get started and i feel that it is more relevant at the beginning because the following steps expect the user to be familiar with docker and the docker link will help achieve the same.